indico/modules/events/abstracts/templates/management/_notification_tpl_list.html

```diff
@@ -152,6 +152,11 @@
     {% endif %}
 
     <div class="toolbar f-j-end hide-if-locked">
+        <div class="group">
+            <a class="i-button" href="{{ url_for('.quick_setup_notifications', event) }}">
+                {% trans %}Quick setup{% endtrans %}
```

Comment:
Should this be hidden/disabled when all notifications are setup? It's a noop in that case

Reply:
yeah, can do it